applications/MeshMovingApplication/python_scripts/ale_fluid_solver.py

```diff
@@ -33,7 +48,7 @@ def __init__(self, model, solver_settings, parallelism):
                 warn_msg += 'switching it on!'
                 KM.Logger.PrintWarning("::[AleFluidSolver]::", warn_msg)
         else:
-            fluid_solver_settings.AddEmptyValue("compute_reactions").SetBool(True)
+            fluid_solver_settings.AddEmptyValue("compute_reactions").SetBool(False)
```

Comment:
The segmentation fault was starting from this statement. However now I realized that the else_statement should be moved to the second if_statement and not the first one. Since it does not make sense to use the else statement when there's a True statement for compute_reactions